terraform/faucet.tf

```diff
@@ -73,6 +73,7 @@ resource "aws_ecs_service" "faucet" {
   task_definition                    = aws_ecs_task_definition.faucet.arn
   desired_count                      = 1
   deployment_minimum_healthy_percent = 0
+  count                              = var.cluster_test_logging ? 0 : 1
```

Comment:
You can't reference `cluster_test_logging` here because it's defined in the FB-specific configs in libra-ops. Why remove the faucet though? It would be good if cluster-test also verified that faucet is working.